scylladb/cpp-driver
# docs: added sphinx

Related issue https://github.com/scylladb/cpp-driver/issues/15

Adds Sphinx and [sphinx-scylla-theme](https://sphinx-theme.scylladb.com/) to the cpp-driver repository.


# How to test this PR

Run ``make preview`` under the docs directory and open http://localhost:5500/ to check if the site is rendering.
